spec/services/merge_requests/update_service_spec.rb

```diff
@@ -13,15 +13,20 @@
 
   describe :execute do
     context "valid params" do
-      before do
-        opts = {
-          title: 'New title',
-          description: 'Also please fix',
-          assignee_id: user2.id,
-          state_event: 'close'
+      let(:opts) do
+        {
+            title: 'New title',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.